internal/orderbook/limitorder.go

```diff
@@ -77,7 +77,7 @@ func CreateLimitOrderMessage(orderRequest models.CreateOrderParams, makerTraits
 		Extension:     "0x",
 	}
 
-	aggregationRouter, err := constants.Get1inchRouterFromChainId(orderRequest.ChainId)
+	aggregationRouter, err := constants.Get1inchRouterFromChainId(chainId)
```

Comment:
Let's move it to web3data.go insidue orderbook, no any sense to have limit order specific funcs here

Reply:
Here is an updated workspace to show the advantages from a discoverability perspective

![image](https://github.com/1inch/1inch-sdk-go/assets/6529295/cf779b6e-0c2e-4cce-ad60-d9f0e6700950)

The `orderbook` directory now holds three top-level files and gives the user clear direction as to what methods will be relevant to them while creating orderbook calls. The `models` directory serves two purposes here: It collapses all models files into one line-item and additionally hides the `validate.go` file which is also not relevant to the user (but must be next to the model files with the current implementation of being functions attached to the structs).

This lets us move all helper functions away from the users' view entirely while they look at what methods are available to call. A similar functional equivalent would be to use private functions without the `internal` directory, but that would still clutter the files that hold the public methods (the ones the user will be testing/using in their project)